integration_tests/tests/rpc/espace/test_debug_rpc.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+from integration_tests.test_framework.test_framework import ConfluxTestFramework
+from integration_tests.test_framework.util import load_contract_metadata
+from integration_tests.conflux.rpc import RpcClient
+from typing import Type
+
+@pytest.fixture(scope="module")
+def framework_class() -> Type[ConfluxTestFramework]:
+    class DefaultFramework(ConfluxTestFramework):
+        def set_test_params(self):
+            self.num_nodes = 1
+            self.conf_parameters["min_native_base_price"] = 10000
+            self.conf_parameters["next_hardfork_transition_height"] = 1
+            self.conf_parameters["next_hardfork_transition_number"] = 1
+            self.conf_parameters["public_evm_rpc_async_apis"] = "\"all\""
+
+        def setup_network(self):
+            self.setup_nodes()
+            self.rpc = RpcClient(self.nodes[0])
+
+    return DefaultFramework
+
+def test_trace_simple_cfx_transfer(ew3, evm_accounts):
+    account = evm_accounts[0]
+    to_address = ew3.eth.account.create().address
+    tx_hash = ew3.eth.send_transaction({
+        "to": to_address,
+        "value": ew3.to_wei(1, "ether"),
+        "from": account.address
+    })
+    ew3.eth.wait_for_transaction_receipt(tx_hash)
+    assert ew3.eth.get_balance(to_address) == ew3.to_wei(1, "ether")
+
+    tx_trace = ew3.manager.request_blocking('debug_traceTransaction', [tx_hash])
+    assert tx_trace['failed'] == False
+    assert tx_trace['gas'] == 21000
+    assert tx_trace['returnValue'] == ''
+    assert len(tx_trace['structLogs']) == 0
+
+def test_trace_deploy_contract(ew3, evm_accounts):
```

Comment:
This function can be refactored as 

```py
@pytest.fixture(scope="module")
def tx_hash(...):
    ...
    h = TokenContract.constructor(account.address).transact()
    ew3.eth.wait_for_transaction_receipt(h)
    return h

@pytest.fixture(scope="module")
def transfer_hash(tx_hash):
    h = token_contract.functions.transfer(to_address, ew3.to_wei(1, "ether")).transact()
    ew3.eth.wait_for_transaction_receipt(h)
    return h
```

Then check_*** can be written as test_xxx with tx_hash or transfer_hash as fixture